packages/fxa-settings/src/pages/Signin/Signin/en.ftl

```diff
@@ -0,0 +1,17 @@
+# <span> is for visual effect
```

Comment:
```suggestion
# Strings within the <span> elements appear as a subheading.
```
A more descriptive comment would b helpful to understand how the string will appear. (Especially since the second part is breaking into a subheading.)